samwise.js

```diff
@@ -6,35 +6,37 @@
  */
```

Comment:
SCRIPT-I like how you've used promise chaining here, it makes the code flow very clear.